src/firebase/firebaseSetup.js

```diff
@@ -51,5 +51,5 @@ const uiConfig = {
   },
 };
 
-export { StyledFirebaseAuth, uiConfig, db, storage };
+export { StyledFirebaseAuth, uiConfig, db, storage, firebase };
```

Comment:
The default export is already firebase. It shouldn't need to be added to this list.